### PR TITLE
Created-Hello-Microverse-Project-for-Code-Review

### PR DESCRIPTION
**Pull request by fernando-silvabr66**
This pull request was created for the Code Review process and to show the linters deployed, the web page, and CSS files.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (HENRYKC24) [inline .github/workflows/linters.yml:48]: This is great:+1: You used the correct linter config file. Keep that up:clap:

**Review by HENRYKC24 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUESTED :recycle: 
Change requested
Hi @fernando-silvabr66 :wave:
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] You did a good job but I think you forgot to add a pull request summary. Adding a good pull request summary tells the reviewer more about the pull request and what to expect. This makes the work easy for everyone and appear very professional.
image
You can [read this](https://www.pullrequest.com/blog/writing-a-great-pull-request-description/) for more information.

- [ ] While you made a frantic effort to make sure you use GitHub Flow, you seemingly forgot to make your branch name descriptive.
image
For example, you can call the branch `created-hello-microverse-project`
Just a descriptive name that would make it clear what the branch is all about. [Read this](https://linuxize.com/post/how-to-rename-local-and-remote-git-branch/) for more information on **how to rename a local and remote git branch**.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @fernando-silvabr66 ,

Good job so far implementing some of the recommended changes by the previous reviewer however, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] [REQUIRED] It will be nice to make your branch name descriptive as recommended by the previous reviewer,  you can actually edit the branch name. Just a descriptive name that would make it clear what the branch is all about (e.g. 'created_sidebar' or 'added_comments_section'). Kindly refer to [this link](https://linuxize.com/post/how-to-rename-local-and-remote-git-branch/) for more guidance.
img

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by fernando-silvabr66** [comment]
Hi Tolu,

This is my Week 1 in Microverse Program and I´m a little bit confused with
Github.

Let me try to put my points:

1- When I received the 1st PR comment (from @HenryKC24) I tried to change
the branch name as requested (and following the steps I received in the
Code Review feedback), but it occurs that Github closed the #1 PR and ask
me to make a new PR (wich is not recommended by the Code Review process).
So, first I tried creating a new PR but it was to keep the 1st PR
(which does not occur) and Github created a new (#2) PR, then I tried to
keep only the 1st, etc.

2- After your review, I went to Github again and found a way to rename the
branch (within Github), but the same close/ask for a new PR occurs.

3- If you go to my Hello-Microverse right now you can see that the branch
name is changed, but the Pull Requests are with 'closed' status, as you can
see below.
[image: image.png]
[image: image.png]
I mean, I don´t know how to fix it once it is an automatic Github process
(When renaming a branch it closes the PR and asks for a new one).

Please, what should I do in this case?

Regards,
Fernando.

**Comment by fernando-silvabr66** [comment]
below are the images I mentioned:

image

image

**Comment by Whoistolu** [comment]
Hello @fernando-silvabr66 ,
I have read your explanations and I am sorry about how you feel. I understand that you have two PRs closed at the moment, let me invite a senior code reviewer to help look into this.
[Roy Mukuye](https://github.com/mke2111) Kindly help look into this.

**Review by mke2111 — COMMENTED** [review]
Hey @fernando-silvabr66,

Yes, this happens once you try to rename a branch. You always have to keep in mind to create meaningful branch names right from the start, or before you create a PR. 

@HENRYKC24 was meant to invalidate this PR right from the start since the branch name should have the name of the feature implemented.
Since he did not invalidate the PR, @Whoistolu  was meant to approve this PR if that was the only issue that was left to fix.

The next step is to create a new 'meaningful' branch and then a new PR and in this new PR, remember to link this old PR so that the next reviewer can APPROVE.

Thank you @Whoistolu for helping here.

> NOTE Next reviewer, Please approve this PR.

**Comment by Whoistolu** [comment]
Thank you for a swift response @mke2111.

@fernando-silvabr66 Apologies concerning this.

Kindly follow the steps stated above by @mke2111 and your project will be approved by the next reviewer.
Happy coding!!!

**Comment by HENRYKC24** [comment]
Noted @mke2111. I will definitely invalidate the next time this happens. Thank you very much for the clarification and support:pray:

**Pull request closed without merging**